superset/utils/core.py

```diff
@@ -866,9 +866,9 @@ def get_or_create_main_db():
     dbobj = get_main_database(db.session)
     if not dbobj:
         dbobj = models.Database(database_name='main')
+        dbobj.expose_in_sqllab = True
```

Comment:
[optional], to make it clear that these are all initialization / creation time args 
```python
models.Database(
    database_name='main',
    expose_in_sqllab=True,
    allow_csv_upload=True,
)
```